### ISSUE TITLE
objects.merge docs don't match implementation

### ISSUE THREAD

**Issue by jaraco** [dgilland/pydash#62]
In particular, docs for merge say:

```
If the callback returns undefined merging will be handled by the method instead.
```

`undefined` in this context appears to be an artifact of a port from lodash... and looking at the implementation, the result of the callback is not considered.

The easy fix is to delete that sentence. Perhaps better would be to implement support for that functionality, perhaps by testing for `is NoValue`, and update the documentation to match that.

**Comment by dgilland** [comment]
Nice catch!

I've fixed the docstring in 2f1f46290a8bc50b8c16098047b87e12829ed531.

I'm not opposed to implementing the callback functionality but I would consider it a breaking change so would need to be included in a major release.

Is that kind of callback functionality something that you would like to see implemented though?

**Comment by jaraco** [comment]
Thanks!

I wouldn't be using that functionality. It does seem worthwhile to have for consistency, but certainly not something I'd fight for.
